tsconfig.json

```
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "target": "es2015",
+    "downlevelIteration": true,
+    "importHelpers": true,
+    "sourceMap": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "allowJs": false,
+    "checkJs": false,
+    "resolveJsonModule": true,
+    "lib": ["dom", "es2015", "es2016", "es2017", "esnext"],
```

nit: Since we aren't building for the browser:
```suggestion
    "lib": ["es2015", "es2016", "es2017", "esnext"],
```